provisioning-workflows/IIB/IIBv9/provisionIIB.properties

```diff
@@ -0,0 +1,23 @@
+IZU_VARIABLE_SUBSTITUTION_ON
+ZOSMF_INSTALL_USER = TW4#
+ZOSMF_RACF_USER = TW4#
+#ZOSMF_INSTANCE = ZOSWFSO1
+#ZOSMF_INSTANCE_RUNASGRP = ZOSWFSOR
+#ZOSMF_PARMLIB = SYS2.ZOSWFSO1.PARMLIB
+#ZOSMF_PROCLIB = SYS2.ZOSWFSO1.STCPROC
+ZOSMF_INSTANCE = ZOSWFX1
+ZOSMF_INSTANCE_RUNASGRP = ZOSWFXR
+ZOSMF_PARMLIB = SYS2.ZOSWFX1.PARMLIB
+ZOSMF_PROCLIB = SYS2.ZOSWFX1.STCPROC
+IIB_MQ_DS_PREFIX = SYS3
+IIB_DB2_DS_PREFIX = SYS3
+IIB_MQ_PATH_new = /usr/lpp/mqm/V9R0M0
+IIB_JAVAHOME = /usr/lpp/java/J7.1_64
+IIB_MQ_PATH = /usr/lpp/mqm/V9R0M0
+IIB_USS_BIN = /usr/lpp/mqsi/V9R0M0/bin
+IIB_STCDFLTGROUP = USSDBMT5
+IIB_STCGROUP = FUSTCBRK
+IIB_JDBC_JARS = /usr/lpp/db2b10/jdbc/classes
+#IIB_PRD_USER = PT5S
+IIB_PRD_USER = PT5X
+IIB_TIV_USER = TIVMANU
```

Comment:
We should include some comments here, and remove all references to site specific settings; much like how sample files today are provided:

> (e.g.)
>`## The "IIB_MQ_DS_PREFIX" parameter indentifies the High Level Qualifier for the MQ Series datasets. 
>IIB_MQ_DS_PREFIX = SYS3

>`## The "ZOSMF_PARMLIB" parameter indentifies which Parmlib member is used.
>ZOSMF_PARMLIB = SYS1.PARMLIB